dookug-api/dookug-api-jee8/dookug-api-rest-jee8/src/main/java/hu/icellmobilsoft/dookug/api/url/IServicePath.java

```diff
@@ -0,0 +1,200 @@
+/*-
+ * #%L
+ * DookuG
+ * %%
+ * Copyright (C) 2023 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.dookug.api.url;
+
+/**
+ * Base http path constants
+ *
+ * @author Imre Scheffer
+ * @since 0.1.0
+ */
+public interface IServicePath {
+
+    /**
+     * Epmpty String
+     */
+    String EMPTY = "";
+    /**
+     * '?' String for http rest paramer concatenation
+     */
+    String SIGN_QUESTION = "?";
+    /**
+     * '=' String for http rest paramer value
+     */
+    String SIGN_EQUALS = "=";
+    /**
+     * &amp; String for http rest paramer concatenation
+     */
+    String SIGN_AND = "&";
+
+    /**
+     * Rest path prefix for public UI (Frontend, Mobile...) category URL
+     */
+    String PREFIX_REST = "/rest";
+
+    /**
+     * Rest path prefix for component inner communication (another inner service) category URL
+     */
+    String PREFIX_SYSTEM = "/system";
+
+    /**
+     * Rest path prefix for public UI and service withput authentication (Frontend, Mobile, API) category URL
+     */
+    String PREFIX_PUBLIC = "/public";
+
+    /**
+     * Rest path prefix for external partner (ticketing partner, toll system...) category URL
+     */
+    String PREFIX_EXTERNAL = "/external";
+
+    /**
+     * Rest path prefix for internal partner service (document service, paying service) category URL
+     */
+    String PREFIX_INTERNAL = "/internal";
+
+    /**
+     * Rest path prefix for internal test service (data generators, flow accelerator) category URL
+     */
+    String PREFIX_TEST = "/test";
+
+    /**
+     * Rest path prefix for maintenance service (for administrators) category URL
+     */
+    String PREFIX_MAINTENANCE = "/maintenance";
+
+    /**
+     * Rest path prefix for customer domain URL
+     */
+    String PREFIX_CUSTOMER = "/customer";
+
+    /**
+     * Rest path prefix for exemption domain URL
+     */
+    String PREFIX_EXEMPTION = "/exemption";
+
+    /**
+     * Rest path prefix for eobu domain URL
+     */
+    String PREFIX_EOBU = "/eobu";
```

Comment:
Ez nagyon fura tartalom

Reply:
jajanyam, toroltem is az osztalyt (masik MR lesz)